index.bs

```diff
@@ -1208,7 +1208,8 @@ run the following steps:
     Note: The exact value of |expires| is not important for the purposes of this algorithm,
     as long as it is in the past.
 
-1. Let |value| be any non-empty [=implementation-defined=] string.
+1. Let |value| be the empty string. 
+1. If |name|'s [=string/length=] is 0, then set |value| to any non-empty [=implementation-defined=] string.
```

Comment:
Final nit: you need to add newlines between the steps to match the existing style of the document.